src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -20,16 +20,18 @@ public class FindCommand extends Command {
             + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
             + "Example: " + COMMAND_WORD + " alice bob charlie";
```

Comment:
Perhaps we need to update this as well!

Reply:
Good catch. I will update it accordingly.